src/Elmish.WPF/ViewModel.fs

```diff
@@ -259,22 +287,23 @@ and internal FuncsFromSubModelSeqKeyed() =
 
 
 and internal Initialize
-      (log: ILogger,
-       logPerformance: ILogger,
-       performanceLogThresholdMs: int,
-       nameChain: string,
-       getNameChainFor: string -> string,
-       getNameChainForItem: string -> string -> string,
+      (loggingArgs: LoggingViewModelArgs,
        name: string,
        getFunctionsForSubModelSelectedItem: string -> ((obj -> obj) * (obj -> ViewModel<obj, obj> option)) option) =
 
+  let { log = log
+        logPerformance = logPerformance
+        performanceLogThresholdMs = performanceLogThresholdMs
+        nameChain = nameChain } =
+    loggingArgs
+
   let measure x = x |> Helpers2.measure logPerformance performanceLogThresholdMs name nameChain
   let measure2 x = x |> Helpers2.measure2 logPerformance performanceLogThresholdMs name nameChain
 
   member _.Base<'model, 'msg>
       (initialModel: 'model,
-       getCurrentModel: unit -> 'model,
        dispatch: 'msg -> unit,
+       getCurrentModel: unit -> 'model,
```

Comment:
Why swapping the order of these arguments?  I am ok with this.  Just curious.

Reply:
Just to keep `initialModel` and `dispatch` next to each other, since they are a part of `ViewModelArgs` (and the other isn't). I was thinking maybe of splitting that part into its own thing, but thought better of it.